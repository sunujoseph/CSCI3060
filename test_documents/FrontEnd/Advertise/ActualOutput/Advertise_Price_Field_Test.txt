path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Advertise\ActualDTF\Advertise_Price_Field_Test.txt
Enter Username: temp: |
advertiseÍÍÍÍÍÍÍııııD„|
username: |userSS
advertis| not found
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
