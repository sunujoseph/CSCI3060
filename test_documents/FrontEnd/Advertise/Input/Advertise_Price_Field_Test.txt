login
userSS
advertise
New Item
99dollars
