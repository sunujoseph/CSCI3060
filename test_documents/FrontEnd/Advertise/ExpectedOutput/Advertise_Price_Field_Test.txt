Enter Username: 
Enter Item Name: 
Enter Minimum Bid: 
Error: Invalid Input
Enter Minimum Bid: 